portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,41 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Lindsey's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+    <div id="introduction">
+      <h1>Hey, I'm Lindsey.</h1>
+      <h2>It's nice to meet you.</h2>
+      <p>This is me.</p>
+      <img src="images/profilepic.JPG"/>
+    </div>
+
+    <div id="projects">
+    <p>Here are some of the projects I've been working on:</p>
+    <ul>
+      <li>CU@Home - an augmented reality snapchat filter that lets you
+      experience school from wherever you are</li>
+      <li>Fractals with Python Turtle - An interactive program that lets
+      you create different fractals of differeing orders</li>
+      <li>Java SpellChecker - checks to see if your word is an a given
+      dictionary and gives you suggestions if it's not</li>
+    </ul>
+    </div>
+
+    <div id="links">
+	    <p>Find me on <a href="https://github.com/LindseyWeiskopf">GitHub</a> and <a href="https://www.linkedin.com/in/lindsey-weiskopf">LinkedIn</a>.</p>
+    </div>
```

Comment:
nit: Please use indentations of 2 spaces at a time. 

https://google.github.io/styleguide/htmlcssguide.html#Indentation